### PR TITLE
el8: add python interpreter

### PR DESCRIPTION
**Pull request by tomaskrizek**
Add python interpreter which is missing in the base installation of EL8
and its derivates. Besides other uses, python interpreter is required
for ansible provisioning.

### CODE REVIEW

**Comment by ladar** [comment]
@tomaskrizek doesn't RHEL ship with `python` by default? The `dnf` tool is written in it! Or so I thought. 

And with v8 there is this new concept of streams. So you can specify whether you want to track the Python 2.7 stream, or the Python 3.6 stream... `dnf module list` gives:

```
python27                     2.7 [d][e]            common [d]                                      Python programming language, version 2.7
python36                     3.6 [d][e]            common [d], build                               Python programming language, version 3.6 
```

Although I don't know what it will show on the box images ... 

I think there is also the `/usr/libexec/platform-python` binary, which is what the system tools use. 

Long story short, I'm not opposed to adding an explicit Python stream to the images, but since it's too late for the 1.9.38 boxes, I'll leave this pull request open for a couple of weeks, so others can voice any opinions they might have.

In particular, I'm thinking we might want to use the "unversioned" stream ... so it tracks whatever Red Hat picks as the default.

**Comment by tomaskrizek** [comment]
- `python3` seems to be an alias to `python36`
- AFAIK `/usr/libexec/platform-python` should only be used by system tools, such as `dnf`, and it is separated from the python packages so the streams won't break base tools
- I wouldn't worry about python2, since it won't be supported by upstream any more since 2020-01-01 https://pythonclock.org/

**Comment by MartinBasti** [comment]
Explicit is better than implicit. I'd go with py3 as everything in centos8 runs on py3. Py2 is there only for compatibility.

**Comment by tomaskrizek** [comment]
@ladar There doesn't seem to be any opposition to including Python3 to the boxes. Could you please merge this?
